x-pack/auditbeat/module/system/system.go

```diff
@@ -55,19 +55,20 @@ func NewModule(base mb.BaseModule) (mb.Module, error) {
 	log := logp.NewLogger(moduleName)
 
 	var hostID string
-	hostInfo, err := sysinfo.Host()
-	if hostInfo != nil {
```

Comment:
What was going on in here is when `sysinfo.Host()` errors, it returns a non-nil interface that wraps a nil pointer, causing the nil check to pass but a panic inside the `hostInfo.Info()` call below.